project/plugins.sbt

```diff
@@ -11,7 +11,7 @@ resolvers += Resolver.url(
   new URL("https://broadinstitute.jfrog.io/broadinstitute/libs-release/")
 )(publishPatterns)
 
-val sbtPluginsVersion = "0.9.0"
+val sbtPluginsVersion = "0.12.0"
```

Comment:
I noticed this value in the clinvar project is "0.12.1"
https://github.com/DataBiosphere/clinvar-ingest/blob/master/project/plugins.sbt#L1
Should the version numbers be the same?

Reply:
Oh good catch! It probably should. @danxmoran do you agree?

Reply:
Yes! Great catch Quazi, 0.12.0 is broken